pkgs/stdenv/generic/setup.sh

```diff
@@ -422,50 +421,55 @@ fi
 
 
 substitute() {
-    local input="$1"
-    local output="$2"
+    local input=$1
+    local output=$2
```

Comment:
This is covered under PARAMETERS in the bash manual. Word splitting & pathname expansion is not performed on assignment.